src/helper.ts

```diff
@@ -75,20 +75,21 @@ export function generateConfigs(configs: ConfigsOptions = {}): string {
     'isNewTab',
     'forceLogout',
     'sdkPlatform',
-    'sdkVersion'
+    'sdkVersion',
+    'authnMethod',
   ];
 
   for (const key in configs) {
     if (configKeys.indexOf(key) !== -1) {
       snakeCaseConfigs[snakeCase(key)] = configs[key as keyof ConfigsOptions];
     }
   }
-
+  
   return stringify({
     sdk_platform: 'js',
     sdk_version: __VERSION__,
     ...snakeCaseConfigs
-  });
+  }, { indices: false, arrayFormat: 'brackets', encode: false });
```

Comment:
`encode: false` is necessary?

Reply:
Yes, because by default `qs` encodes the brackets but we expect an un-encoded string from that function

Reply:
will it breaks `back_to` and `email` in this case?

Reply:
We added a test to ensure the configs are encoding special characters